### ISSUE TITLE
Closure library deleted goog.net.jsloader.load

### ISSUE THREAD

**Issue by grinderrz** [adzerk-oss/boot-reload#121]
Which results in  

```
cljs.user=> (require '[my.namespace :as l] :reload)
#object[TypeError TypeError: goog.net.jsloader.load is not a function]

TypeError: goog.net.jsloader.load is not a function
    at adzerk.boot_reload.client.patch_goog_base_BANG_.goog.global.CLOSURE_IMPORT_SCRIPT (ht
tp://localhost:3000/js/main.out/adzerk/boot_reload/client.js:18:26)
    at Object.goog.importScript_ (http://localhost:3000/js/main.out/goog/base.js:951:9)
    at Object.goog.writeScripts_ (http://localhost:3000/js/main.out/goog/base.js:1394:16)
    at Object.goog.require [as require__] (http://localhost:3000/js/main.out/goog/base.js:70
6:14)
    at Object.clojure.browser.repl.bootstrap.goog.require (http://localhost:3000/js/main.out
/clojure/browser/repl.js:295:16)
    at eval (eval at <anonymous> (http://localhost:3000/js/main.out/weasel/repl.js:30:495),
<anonymous>:1:6)
    at http://localhost:3000/js/main.out/weasel/repl.js:30:495
    at http://localhost:3000/js/main.out/weasel/repl.js:39:4
    at cljs.core.MultiFn.G__12183__2 (http://localhost:3000/js/main.out/cljs/core.js:34490:1
06)
    at cljs.core.MultiFn.G__12183 [as call] (http://localhost:3000/js/main.out/cljs/core.js:
34757:20)
``` 

According [this commit](https://github.com/google/closure-library/commit/988ff900886e959a888485ee56d1540d1bf2f29d), one should use .safeLoad instead.

**Comment by Deraen** [comment]
This was fixed in 0.5.2.

**Comment by grinderrz** [comment]
Experiencing this on 0.5.2 now.
As I understand, there was fixed a similar issue #117 with .loadMany
Call to .load is still [here](https://github.com/adzerk-oss/boot-reload/blob/2b0ba8b0e27e38e02b66711dc141b4866cf31b00/src/adzerk/boot_reload/client.cljs#L19)

**Comment by Kpovoc** [comment]
I am also having a similar error, and can confirm that the following from client.cljs seems to be the trouble code.
```cljs
; Thanks, lein-figwheel & lively!
(defn patch-goog-base! []
  (set! (.-provide js/goog) (.-exportPath_ js/goog))
  (set! (.-CLOSURE_IMPORT_SCRIPT (.-global js/goog)) (fn [file]
                                                       (when (.inHtmlDocument_ js/goog)
                                                         (jsloader/load file)))))
```

**Comment by Deraen** [comment]
Does this happen when using repl?

**Comment by Kpovoc** [comment]
Yes

**Comment by oakes** [comment]
FWIW this seemed to happen to me when I tried directly running `client/connect` at the top level of an ns that was in my `:preloads`. When I moved it into my window.onload function, the problem went away. Not sure what to make of that, but hopefully it helps.

**Comment by ikzk** [comment]
Also experiencing this

**Comment by ska2342** [comment]
Updating from 0.5.1 to 0.5.2 fixed the problem for me,

**Comment by Deraen** [comment]
https://github.com/adzerk-oss/boot-reload/pull/122 has been merged and released in 0.6.0
